### ISSUE TITLE
skipping a vehicle with persons

### ISSUE THREAD

**Issue by behrisch** [eclipse-sumo/sumo#9026]
if a vehicle is defined with "triggered" persons such that it starts with the people inside it is not possible to skip it by defining a begin time after the departure time of the vehicle. The triggered persons throw an error for an unknown vehicle.

**Comment by behrisch** [comment]
The only possible way to fix this is probably to track all skipped vehicles. @namdre Ideas?

**Comment by namdre** [comment]
Wouldn't it work if the persons are moved to the correct part of the input or the vehicle is loaded from an additional file?

**Comment by behrisch** [comment]
No, at least in my example the vehicle is in an additional file and the error is still thrown.

**Comment by namdre** [comment]
would also be fixed via #12616 (though potentially with some reduction in efficiency since there will still be repeated attempts to find a suitable insertion vehicle).
